docs/source/code/cancer_schema.tsv

```diff
@@ -5,12 +5,12 @@ title           Example matched cancer tumor/normal study
 description     The study has two patients, P001 has one tumor sample, P002 has two
```

Comment:
i did, but this breaks the rendering in html pages.    

i think this should be reverted to space delimiting